BookingsApi/BookingsApi.Infrastructure.Services/EventDtoMappers.cs

```diff
@@ -54,7 +54,7 @@ public static HearingConfirmationForParticipantDto MapToHearingConfirmationDto(G
                 DisplayName = string.IsNullOrEmpty(participant.DisplayName) ? $"{participant.JudiciaryPerson.KnownAs} {participant.JudiciaryPerson.Surname}" : participant.DisplayName,
                 Representee = "",
                 Username = participant.JudiciaryPerson.Email, // we need to pass a username otherwise the notification is failing 
-                ContactEmail = participant.JudiciaryPerson.Email,
+                ContactEmail = participant.ContactEmail,
```

Comment:
I realize what's happened here, we should never be accessing the contactEmaill, directly via the the domain object property, we should be using the GetEmail() accessor method, which will return correctly the contact email or regular email if that has not been populated

Reply:
Ideally we would make the contact email property private, but because it's a property on an entity, it needs to be public for entity framework to be able to read it

Reply:
done